src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -352,4 +352,6 @@ function editCourse() {
 
 $(document).ready(function() {
     $('#courseEditLink').click(editCourse);
+    var index = $('#new-instructor-index').val();
```

Comment:
Yes it would be good to standardise the ID naming convention throughout the code. Do go ahead and create an issue for it